SAPHana/man/SAPHanaSrMultiTarget.py.7

```diff
@@ -1,6 +1,6 @@
 .\" Version: 0.180.0
 .\"
-.TH SAPHanaSrMultiTarget.py 7 "27 Apr 2023" "" "SAPHanaSR-ScaleOut"
+.TH SAPHanaSrMultiTarget.py 7 "04 Jan 2023" "" "SAPHanaSR-ScaleOut"
```

Comment:
can you please adapt the year too even that traveling back in time may be exciting :smile:

Reply:
done